src/api/middleware/validators/offer.js

```diff
@@ -166,7 +188,7 @@ const jobMaxDurationEditable = async (jobMaxDurationCandidate, { req }) => {
         const { jobMinDuration: jobMinDurationCandidate } = req.body;
 
         const jobMinDuration = jobMinDurationCandidate || offer.jobMinDuration.toString();
-        if (jobMinDuration >= jobMaxDurationCandidate) {
+        if (jobMinDuration > jobMaxDurationCandidate) {
```

Comment:
Add a check before this `if (!jobMinDuration) throw...` since jobMinDuration is required if jobMaxDuration is present

Reply:
Yes, I didn't thought of this problem. 

In that case, maybe we should throw an error saying something like "Must add minimum offer duration" or just assume it is 0, in that case it should be accepted.

Reply:
Since there is no default 0 on the model side, let's not assume 0 and just throw the error

Reply:
@DoStini I reused the same error I was using in the creation validator (JOB_MIN_DURATION_NOT_SPECIFIED)